application/app.py

```diff
@@ -549,17 +544,17 @@ def saveConfig_objects(self):
             update_config_with_sections(config_path, "config", "mm-segmentation-distance", mm_segmentation_distance)
         else: mm_segmentation_distance = None
 
-        self.api.configFiles(get_config_with_sections(get_config_path(), 'info', 'identifier'),\
+        api.configFiles(get_identifier(),\
                      max_connection_distance = mm_connection_distance,\
                      max_segmentation_distance = mm_segmentation_distance)
 
     def loadConfig_objects(self):
         config_path = get_config_path()
 
-        exists, frame1 = get_config_with_sections(config_path, "config", "frame1")
-        exists, nframes = get_config_with_sections(config_path, "config", "nframes")
-        exists, mm_connection_distance = get_config_with_sections(config_path, "config", "mm-connection-distance")
-        exists, mm_segmentation_distance = get_config_with_sections(config_path, "config", "mm-segmentation-distance")
+        frame1 = get_config_with_sections(config_path, "config", "frame1")
+        nframes = get_config_with_sections(config_path, "config", "nframes")
```

Comment:
I think these were changed to start_frame and num_frames in dev? I will resolve the conflict as such.